File: api/riskfunction/get.js
// Load the AWS SDK for Node.js
const AWS = require('aws-sdk')
// Set region
AWS.config.update({region: 'us-west-2'})
// Create S3 service
const S3 = new AWS.S3({apiVersion: '2006-03-01'})
// Define headers
const headers = {
  "Content-Type": "application/json",
  "Access-Control-Allow-Origin": "*",
  "Access-Control-Allow-Headers": "Content-Type",
  "Access-Control-Allow-Methods": "OPTIONS,GET,POST,PUT,DELETE"
}
// Lambda handler
exports.handler = async function(event) {
  return new Promise((resolve, reject) => {
    resolve({
      statusCode: 200,
      headers: headers,
      body: "data goes here"
    })
  })
}

File: api/riskfunction/get.py
from api.utils import *

def risk_function():
    return json.dumps({
        "results": "go here"
    })

def handler(event, context):
    try:
        body = risk_function()
        return {
            "statusCode": 200,
            "body": body,
            "headers": headers
        }
    except InputException as e:
        return handle_error(event, e, 400)
    except Exception as e:
        return handle_error(event, e, 500)
